airbyte-integrations/connectors/source-pinterest/source_pinterest/manifest.yaml

```diff
@@ -125,6 +125,229 @@ definitions:
+            - PAID_IMPRESSION
+            - PIN_ID
+            - PIN_PROMOTION_ID
+            - REPIN_1
+            - REPIN_2
+            - REPIN_RATE
+            - SPEND_IN_DOLLAR
+            - SPEND_IN_MICRO_DOLLAR
+            - TOTAL_CHECKOUT
+            - TOTAL_CHECKOUT_VALUE_IN_MICRO_DOLLAR
+            - TOTAL_CLICKTHROUGH
+            - TOTAL_CLICK_ADD_TO_CART
+            - TOTAL_CLICK_CHECKOUT
+            - TOTAL_CLICK_CHECKOUT_VALUE_IN_MICRO_DOLLAR
+            - TOTAL_CLICK_LEAD
+            - TOTAL_CLICK_SIGNUP
+            - TOTAL_CLICK_SIGNUP_VALUE_IN_MICRO_DOLLAR
+            - TOTAL_CONVERSIONS
+            - TOTAL_CUSTOM
+            - TOTAL_ENGAGEMENT
+            - TOTAL_ENGAGEMENT_CHECKOUT
+            - TOTAL_ENGAGEMENT_CHECKOUT_VALUE_IN_MICRO_DOLLAR
+            - TOTAL_ENGAGEMENT_LEAD
+            - TOTAL_ENGAGEMENT_SIGNUP
+            - TOTAL_ENGAGEMENT_SIGNUP_VALUE_IN_MICRO_DOLLAR
+            - TOTAL_IDEA_PIN_PRODUCT_TAG_VISIT
+            - TOTAL_IMPRESSION_FREQUENCY
+            - TOTAL_IMPRESSION_USER
+            - TOTAL_LEAD
+            - TOTAL_OFFLINE_CHECKOUT
+            - TOTAL_PAGE_VISIT
+            - TOTAL_REPIN_RATE
+            - TOTAL_SIGNUP
+            - TOTAL_SIGNUP_VALUE_IN_MICRO_DOLLAR
+            - TOTAL_VIDEO_3SEC_VIEWS
+            - TOTAL_VIDEO_AVG_WATCHTIME_IN_SECOND
+            - TOTAL_VIDEO_MRC_VIEWS
+            - TOTAL_VIDEO_P0_COMBINED
+            - TOTAL_VIDEO_P100_COMPLETE
+            - TOTAL_VIDEO_P25_COMBINED
+            - TOTAL_VIDEO_P50_COMBINED
+            - TOTAL_VIDEO_P75_COMBINED
+            - TOTAL_VIDEO_P95_COMBINED
+            - TOTAL_VIEW_ADD_TO_CART
+            - TOTAL_VIEW_CHECKOUT
+            - TOTAL_VIEW_CHECKOUT_VALUE_IN_MICRO_DOLLAR
+            - TOTAL_VIEW_LEAD
+            - TOTAL_VIEW_SIGNUP
+            - TOTAL_VIEW_SIGNUP_VALUE_IN_MICRO_DOLLAR
+            - TOTAL_WEB_CHECKOUT
+            - TOTAL_WEB_CHECKOUT_VALUE_IN_MICRO_DOLLAR
+            - TOTAL_WEB_CLICK_CHECKOUT
+            - TOTAL_WEB_CLICK_CHECKOUT_VALUE_IN_MICRO_DOLLAR
+            - TOTAL_WEB_ENGAGEMENT_CHECKOUT
+            - TOTAL_WEB_ENGAGEMENT_CHECKOUT_VALUE_IN_MICRO_DOLLAR
+            - TOTAL_WEB_SESSIONS
+            - TOTAL_WEB_VIEW_CHECKOUT
+            - TOTAL_WEB_VIEW_CHECKOUT_VALUE_IN_MICRO_DOLLAR
+            - VIDEO_3SEC_VIEWS_2
+            - VIDEO_LENGTH
+            - VIDEO_MRC_VIEWS_2
+            - VIDEO_P0_COMBINED_2
+            - VIDEO_P100_COMPLETE_2
+            - VIDEO_P25_COMBINED_2
+            - VIDEO_P50_COMBINED_2
+            - VIDEO_P75_COMBINED_2
+            - VIDEO_P95_COMBINED_2
+            - WEB_CHECKOUT_COST_PER_ACTION
+            - WEB_CHECKOUT_ROAS
+            - WEB_SESSIONS_1
+            - WEB_SESSIONS_2
+          level: "{{ parameters['level'] }}"
+    download_target_extractor:
+      type: DpathExtractor
+      field_path:
+        - url
+    record_selector:
+      type: RecordSelector
+      extractor:
```

Comment:
could you please add` schema_normalization: Default` to record selector, so regression tests will not fail with errors like `['TOTAL_VIDEO_P100_COMPLETE']['type'] changed from "number" to "integer".`